### PR TITLE
objects sorting

### PR DESCRIPTION
**Pull request by mohitpubnub**
@crimsonred  Unit tests editmode are passing on local machine. Please review code so I can solve  unit test editmode failure issue reported by travis

### CODE REVIEW

File: PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeTests.cs [hunk 1]
```diff
@@ -848,4666 +923,5430 @@ public IEnumerator TestHereNowEmptyChannel()
...
-							Assert.IsTrue(!found1);
-							Assert.IsTrue(!found2);
-							tresult = !found1 && !found2;
-						}
+            yield return new WaitForSeconds(PlayModeCommon.WaitTimeBetweenCalls4);
+            Assert.True(tresult, "GetMembershipsFilter didn't return");
+
+            // Sort feature test for GetMemberships
+            tresult = false;
+            pubnub.GetMemberships().UserID(userid).Include(inclMem).Sort(sort).Limit(1).Count(count).Async((result, status) =>
```
Comment (crimsonred) [inline PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeTests.cs]: @mohitpubnub please enhance the test to include that it's actually sorting the response.

**Comment by crimsonred** [comment]
@mohitpubnub the PR should be for develop

**Comment by crimsonred** [comment]
@mohitpubnub please resolve conflicts

File: PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeTests.cs (hunk 1 above)
Comment (mohitpubnub) [reply PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeTests.cs]: Yes, I should. So I am adding this to my TODO list

**Comment by mohitpubnub** [comment]
creating new one with clean code/commits with implemented comments

**Pull request closed without merging**